wallet/wallet.go

```diff
@@ -3646,8 +3646,13 @@ func (w *Wallet) publishTransaction(tx *wire.MsgTx) (*chainhash.Hash, error) {
 	var (
 		txid      = tx.TxHash()
 		returnErr error
+		txRaw     bytes.Buffer
 	)
 
+	// The serialized transaction is for logging only, don't fail on the
+	// error.
+	_ = tx.Serialize(&txRaw)
```

Comment:
Not sure how costly is this `Serialize` method, but it seems we can call it at L3822 to save a few CPU cycles.

Reply:
Good point, done.